docs/multicluster/user-guide.md

```diff
@@ -744,23 +655,23 @@ spec:
       matchLabels:
         app: db
   priority: 1
-  tier: application 
+  tier: application
   ingress:
   - action: Allow
     from:
-    # Select all Pods in Namespace "prod-us-west" from all clusters in the ClusterSet (if the 
+    # Select all Pods in Namespace "prod-us-west" from all clusters in the ClusterSet (if the
     # Namespace exists in that cluster) whose labels match app=client
     - scope: clusterSet
-      podSelector:        
+      podSelector:
         matchLabels:
           app: client
   - action: Deny
 ```
 
-As shown in the examples above, setting `scope` to `clusterSet` expands the scope of the
-`podSelector` or `namespaceSelector` of an ingress peer to the entire ClusterSet that the
-policy is created in. Similar to Egress NetworkPolicy, the scope of `appliedTo`
-is also restricted to the local cluster.
+As shown in the examples above, setting `scope` to `clusterSet` expands the
```

Comment:
It's not introduced by this change, but I noticed that the valid scope is `Cluster` and `ClusterSet` as blow, `clusterSet` is not valid anymore. Do you think it's OK to update `clusterSet` to `ClusterSet` here and the sample manifests in this PR? thanks.
```
                            scope:
                              type: string
                              enum: ['Cluster', 'ClusterSet']
```

Reply:
I guess so. @Dyanngg @GraysonWu : please confirm.

Reply:
Yes `Cluster` and `ClusterSet` should be captalized.